content/departments/engineering/adding-to-roadmap.md

```diff
@@ -33,12 +33,12 @@ We don't use a formal/complicated framework for evaluating impact for now, in or
 
 That said, here’s a rough guideline for evaluating impact of customer-facing features:
 
-- **High**: Large ARR (>$5M ARR) or MAU (> 500 DAUs) impact, in the next 12 months. Fit with strategy. Well validated. Differentiated feature. Eg. a project like Batch Changes, Insights, Own, or adding support for a broadly-used codehost.
-- **Medium**: Large ARR (>$1M ARR) or MAU (> 100 DAUs) impact, in the next 12 months. Fit with strategy. Elements of validation.
+- **High**: Large ARR (>$5M ARR) or DAU (> 500 DAUs) impact, in the next 12 months. Fit with strategy. Well validated. Differentiated feature. Eg. a project like Batch Changes, Insights, Own, or adding support for a broadly-used codehost.
+- **Medium**: Large ARR (>$1M ARR) or DAU (> 100 DAUs) impact, in the next 12 months. Fit with strategy. Elements of validation.
```

Comment:
@malomarrec - I thought we stop using MAUs altogether? Also seems like a typo by using "> X DAUs" to number MAU?